src/main/java/by/itech/lab/supplier/dto/mapper/WarehouseMapper.java

```diff
@@ -35,8 +34,15 @@ public WarehouseDto map(Warehouse entity) {
                 .type(entity.getType())
                 .totalCapacity(entity.getTotalCapacity())
                 .address(entity.getAddress())
-                .customerDto(customerMapper.map(entity.getCustomer()))
                 .usersDto(entity.getUsers().stream().map(userMapper::map).collect(Collectors.toSet()))
                 .build();
     }
-}
+
+    public void map(final WarehouseDto from, final Warehouse to) {
+        to.setIdentifier(from.getIdentifier());
+        to.setType(from.getType());
+        to.setTotalCapacity(from.getTotalCapacity());
+        to.setAddress(from.getAddress());
+        to.setUsers(from.getUsersDto().stream().map(userMapper::map).collect(Collectors.toSet()));
```

Comment:
Can UsersDto be null here?

Reply:
we discussed this, and make decision that we don't need this user at all. Thanks for bringing this to my attention